osu.Game/Overlays/RankingsOverlay.cs

```diff
@@ -184,12 +185,12 @@ private void createScoreTable()
 
         private void loadTable(Drawable table)
         {
-            LoadComponentAsync(table, t =>
+            showTableDelegate = Schedule(() => LoadComponentAsync(table, t =>
```

Comment:
could you explain how the schedule helps here?

Reply:
I thought this is what this comment is about
>I'm not 100% certain this cancellation logic is bug-free due to the shared field. I'll have to look in detail in an IDE, but if this is captured as a closure instead of just cancellationToken this might result in races.